label_studio_sdk/project.py

```diff
@@ -6,12 +6,16 @@
 
 from enum import Enum, auto
 from random import sample, shuffle
-from requests.exceptions import HTTPError
+from requests.exceptions import HTTPError, InvalidSchema, MissingSchema
 from pathlib import Path
 from typing import Optional, Union, List, Dict, Callable
 from .client import Client
 from .utils import parse_config
 
+from label_studio_tools.core.utils.io import get_local_path
+from label_studio_tools.core.label_config import parse_config
```

Comment:
Is `from .utils import parse_config` a deprecated one? Maybe we can remove it and use a function from `label-studio-tools` over all SDK?

Reply:
Fixed